### PR TITLE
Added 1.5 draft / 1.6 Challenge

### PR DESCRIPTION
**Pull request by firecrackerlinux**

### CODE REVIEW

File: 1.5 Data Manipulation/data.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+How about you pause this video and give it a go making setters for stamina and damage output?
+
+___
+
+Great! If you had any trouble, post below in the comments. Here's a quick example of how these setters can now be used.
+
+```
+public void PoisonPotion(Character target)
+{
+    target.SetHealth( target.getHealth() - 10 );
```
Comment (antoinegag) [inline 1.5 Data Manipulation/data.md:43]: What if `target` is `null` ?

Comment (antoinegag) [inline 1.5 Data Manipulation/data.md:43]: Would add some check to make sure HP does not fall under 0.

Comment (firecrackerlinux) [reply 1.5 Data Manipulation/data.md:43]: These are hypotheticals meant to illustrate what could be achieved with some imagination, but I'll add that in.

**Review by firecrackerlinux — COMMENTED** [review]
Edited explanation of the poison errors.

**Pull request merged**